mmv1/third_party/terraform/go.mod.erb

```diff
@@ -5,7 +5,7 @@ go 1.18
 
 require (
 	cloud.google.com/go/bigtable v1.16.0
-	github.com/GoogleCloudPlatform/declarative-resource-client-library v1.19.0
+	github.com/GoogleCloudPlatform/declarative-resource-client-library v1.21.1
```

Comment:
Ah- with `1.21.1` there are some changes to other resources included. Can you ensure the DCL build gardener updates this, in a separate PR? go/dcl-build-gardener.

We'll need to keep the changes around for now, so that this PR works, but rebase them out once DCL's updated it. Otherwise we need to manage changelogs and review for other changes in the same PR, which is hectic (and would make you responsible for them)

Reply:
Acknowledged, will do so

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/6529 Here is the PR I made up to upgrade the DCL to v1.21.1